parts/k8s/cloud-init/artifacts/cse_config.sh

```diff
@@ -311,6 +311,12 @@ installContainerd() {
     apt_get_update || exit 99
     apt_get_install 20 30 120 moby-runc moby-containerd=${CONTAINERD_VERSION}* --allow-downgrades || exit 27
   fi
+  CUSTOM_CONTAINERD_URL={{GetCustomContainerdURL}}
```

Comment:
Let's wrap all of these lines in a new template func like `{{HasCustomLinuxContainerdURL}}` so that we don't ship these bytes in all the normal scenarios where we're not using a custom URL.